vtex-apps/admin-pages
# Allow disabling the "Add to Home Screen" prompt

### Depends on vtex/pwa-graphql#29

#### What is the purpose of this pull request?

As the title says. We need this because not every user wants it's PWA to be installed. Some users have their own native app, for example.

#### What problem is this solving?

Users with native apps now can disable the PWA installation (Add to Home Screen) without disabling the whole PWA and its features, like service worker.

#### How should this be manually tested?

[Access the admin](https://pwa--storecomponents.myvtex.com/admin/cms/storefront), change the "Add to Home Screen" toggle and save the PWA settings. Then, [see the worker](https://pwa--storecomponents.myvtex.com/pwa/workers/sw.js?v=0) (you may need to change the query params to see the difference)

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/15948386/55797206-36141b80-5aa2-11e9-8bcd-15687b421f7e.png)

![image](https://user-images.githubusercontent.com/15948386/55797267-5e037f00-5aa2-11e9-9b63-0b073ec3eb5d.png)

![image](https://user-images.githubusercontent.com/15948386/55797364-930fd180-5aa2-11e9-9c7e-90172a23cd99.png)

#### Types of changes
